lib/init/config-initializer.js

```diff
@@ -427,8 +433,7 @@ function promptUser() {
                 { message: "XO: https://github.com/xojs/eslint-config-xo", name: "xo" }
             ],
             skip() {
-                this.state.answers.packageJsonExists = npmUtils.checkPackageJson();
-                return !(this.state.answers.source === "guide" && this.state.answers.packageJsonExists);
+                return !(this.state.answers.source === "guide" && packageJsonExists);
```

Comment:
if packageJsonExists is alway `true`, so the check seems no longer needed.
```suggestion
                return !(this.state.answers.source === "guide");
```

Reply:
Thanks. I forgot to remove these 😅